### DISCUSSION TITLE
Caching Issue

### DISCUSSION THREAD

**Discussion by Ertuss** [reduxjs/redux-toolkit#3700] [Q&A]
Hello, i have a caching issue. We are using server side caching. But backend guy sure himself. Can redux toolkit query cache still store the data after refresh(f5). If it can be happend. I will ask new question.

**Comment by Ertuss** [comment]
The main problem is the user logged out and i am logging in with the new user. I am sure token is right but response is wrong. Also i call 

dispatch(api.util.resetApiState())  when logged out. One more thing whenever i clear cache (CTRL + F5) true response starts come.

**Reply by phryneas** [reply]
RTK Query is an in-memory cache, unless you deliberately persist things, it will always be fresh on a reload.
Maybe this is your browser cache interfering, but you'd see that in the browser Network devtools.
